### PR TITLE
Articles

### PR DESCRIPTION
**Pull request by Michelle-Lohwt**

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -1,4 +1,5 @@
 Rails.application.routes.draw do
   root "pages#home"
   get "about", to: "pages#about"
+  resources :articles, only: [:show]
```
Comment (Michelle-Lohwt) [inline config/routes.rb]: Create routes for `articles`, by default there are 7 routes:
image
and we only choose to keep the `show` action

File: app/controllers/articles_controller.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+class ArticlesController < ApplicationController
+  def show
+    @article = Article.find(params[:id])
```
Comment (Michelle-Lohwt) [inline app/controllers/articles_controller.rb:3]: find the article id from the `articles` database through the corresponding `Article` model and pass it to variable `@article`

File: app/views/articles/show.html.erb [hunk 3]
```diff
@@ -0,0 +1,4 @@
+<h1>Showing article details</h1>
+
+<p><strong>Title: </strong><%= @article.title %></p>
```
Comment (Michelle-Lohwt) [inline app/views/articles/show.html.erb:3]: `<% %>` embeds the ruby code while `=` indicates rendering the ruby code in the `<% %>` 

The `@article` corresponds to the `@article` in the `controllers/articles_controller.rb`

File: app/controllers/articles_controller.rb (hunk 2 above)
Comment (tlkiong) [reply app/controllers/articles_controller.rb:3]: What's the SQL for this?
*assuming you are using a RDBMS

Comment (Michelle-Lohwt) [reply app/controllers/articles_controller.rb:3]: SELECT ID, TITLE, DESCRIPTION
FROM ARTICLES
WHERE ID = params[:id]

Comment (tlkiong) [reply app/controllers/articles_controller.rb:3]: To be more specific, this query is more of `SELECT * FROM articles WHERE id=params[:id]`

**Review by tlkiong — APPROVED** [review]

**Comment by tlkiong** [comment]
image

Hello. Please clean up. Else, it'll be flooded with branches that is not used

**Pull request merged**